svsim/src/main/scala/vcs/Backend.scala

```diff
@@ -50,12 +50,16 @@ object Backend {
     }
 
     final object TraceSettings {
-      final case class FsdbSettings(verdiHome: String)
+      final case class FsdbSettings(
+        verdiHome: String,
+        filename:  String = ""
+      )
```

Comment:
It's not great that both `FsdbSettings` and `TraceSettings` have a `filename`. This can be cleaned up by promoting the trace file stem to a `CommonSimulationSetting`.